std/conv.d

```diff
@@ -2862,7 +2862,19 @@ unittest
 
     // min and max
     real r = to!real(to!string(real.min_normal));
-    assert(to!string(r) == to!string(real.min_normal));
+    version(NetBSD)
+    {
+        // NetBSD notice
+        // to!string returns 3.3621e-4932L. It is less than real.min_normal and it is subnormal value
+        // Simple C code
+        //     long double rd = 3.3621e-4932L;
+        //     printf("%Le\n", rd);
+        // has unexpected result: 1.681050e-4932
```

Comment:
I'm guessing that you missed my comment on this when it got hidden after you changed the code, but please report this in bugzilla and reference the bug number here. Otherwise, it just risks being forgotten and won't get fixed.